bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/messages/Responses/RDLearnedClientsResponse.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.enocean.internal.messages.Responses;
+
+import org.openhab.binding.enocean.internal.Helper;
+import org.openhab.binding.enocean.internal.messages.Response;
+
+/**
+ *
+ * @author Daniel Weber - Initial contribution
+ */
+public class RDLearnedClientsResponse extends Response {
+
+    public class LearnedClient {
+        public byte[] clientId;
+        public byte[] controllerId;
+        public int mailboxIndex;
+    }
+
+    LearnedClient[] learnedClients;
+
+    public RDLearnedClientsResponse(Response response) {
+        this(response.getPayload().length, response.getOptionalPayload().length,
+                Helper.concatAll(response.getPayload(), response.getOptionalPayload()));
+    }
+
+    RDLearnedClientsResponse(int dataLength, int optionalDataLength, byte[] payload) {
+        super(dataLength, optionalDataLength, payload);
+
+        if (payload == null || ((payload.length - 1) % 9) != 0) {
+            return;
+        } else {
+            _isValid = true;
+        }
+
+        learnedClients = new LearnedClient[(payload.length - 1) / 9];
+        for (int i = 0; i < learnedClients.length; i++) {
+            LearnedClient client = new LearnedClient();
+            client.clientId = java.util.Arrays.copyOfRange(payload, 1 + i * 9, 1 + i * 9 + 4);
```

Comment:
Please add `java.util` to the imports and remove the package here.
```suggestion
            client.clientId = Arrays.copyOfRange(payload, 1 + i * 9, 1 + i * 9 + 4);
```

Reply:
fixed